__tests__/app.test.js

```diff
@@ -84,4 +83,44 @@ describe('get api',()=>{
         expect(body).toEqual({'endpoints': endPoints})
       })
   });
-})
\ No newline at end of file
+})
+
+describe('get articles',()=>{
+  test("return 200 and api articles with body property removed", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({body})=>{
+        expect(body.articles).toHaveLength(13);
+            body.articles.forEach((article) => {
+              expect(article).toHaveProperty("author", expect.any(String)),
+              expect(article).toHaveProperty("title", expect.any(String)),
+              expect(article).toHaveProperty("article_id", expect.any(Number)),
+              expect(article).toHaveProperty("topic", expect.any(String)),
+              expect(article).toHaveProperty("created_at", expect.any(String)),
+              expect(article).toHaveProperty("votes", expect.any(Number)),
+              expect(article).toHaveProperty("article_img_url", expect.any(String)),
+              expect(article).toHaveProperty("comment_count", expect.any(String)),
+              expect(article).not.toHaveProperty('body')
+            })
+      })
+  });
+  test("return 404, if non-existant address has been entered", () => {
```

Comment:
In our server, we only need one bad path test. This should not be inside this describe block as it has nothing to do with this endpoint and doesn't access the controller or model for this endpoint. There is no functional different between /api/article and /api/banana :)
Please delete if you already have a bad path test, or if you're adding it for the first time now just pop it in its own describe block.